defAmplitude : 10.0
phaseShift : 0.0
verTraslation : 0.0
